### PR TITLE
Attendance CRU + Search

### PR DESCRIPTION
**Pull request by ManojNathIC**

### CODE REVIEW

**Comment by coolbung** [comment]
Don't commit any image files, remove the file from the uploads folder

File: src/attendance/attendance.controller.ts [hunk 1]
```diff
@@ -1,13 +1,130 @@
...
+  })
+  public async getAttendance(
+    @Param("id") attendanceId: string,
+    @Req() request: Request
+  ) {
+    return this.service.getAttendance(attendanceId, request);
+  }
+
+  @Post()
+  @ApiConsumes("multipart/form-data")
```
Comment (coolbung) [inline src/attendance/attendance.controller.ts:53]: Am merging this for now, so integration can begin. However, the implementation might change, based on the outcome of this discussion https://github.com/shiksha-platform/shiksha-community/discussions/20

File: src/main.ts [hunk 2]
```diff
@@ -2,10 +2,11 @@ import { NestFactory } from "@nestjs/core";
 import { AppModule } from "./app.module";
 import { SwaggerModule, DocumentBuilder } from "@nestjs/swagger";
 import { RequestMethod } from "@nestjs/common";
-
+import { join } from "path";
+import express = require("express");
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-
+  app.use("/uploads", express.static(join(__dirname, "..", "uploads")));
```
Comment (coolbung) [inline src/main.ts:9]: Don't hardcode the path, use it from an environment variable.

Comment (ManojNathIC) [reply src/main.ts:9]: ok sir

**Pull request merged**